### PR TITLE
Fixes to is functions including jQuery's each

### PR DESCRIPTION
**Pull request by softwarespot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -744,7 +744,7 @@ Most of utilities are found by native API. Others advanced functions could be ch
 
   // Native
   function isWindow(obj) {
-    return obj != null && obj === obj.window;
+    return obj !== null obj !== undefined && obj === obj.window;
```
Comment (camsong) [inline README.md]: `obj != null` is short for `obj !== null obj !== undefined`, and the only occasion to use `==`

Comment (softwarespot) [inline README.md]: I would argue in favour of always being explicit no matter what.

Comment (camsong) [inline README.md]: I know your concern, but airbnb style does support eqeq null check https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/best-practices.js#L44
http://eslint.org/docs/rules/no-eq-null

Comment (softwarespot) [inline README.md]: That means it's not supported, because it's set to zero which means an error.

Comment (camsong) [inline README.md]: yep, your are right, can you fix the conflicts?

Comment (softwarespot) [inline README.md]: All done. Thanks for listening to me :)

**Pull request merged**